Extensions/fonts/xrender-fonts.lisp

```diff
@@ -26,6 +26,8 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(setq clim:*default-server-path* '(:clx :font-renderer mcclim-truetype:truetype-font-renderer))
```

Comment:
what if the user have clim:*default-server-path* set to something like (:clx :mirroring :foo :host "/bla/bla") ?

Reply:
You're right that indiscriminately overwriting the server path night not be the best idea. Aternative proposals would be appreciated.